### PR TITLE
Create uliege-droit.csl

### PR DESCRIPTION
**Pull request by graulich**
- New citation style adapted from Université libre de Bruxelles - Histoire (French)
- citation style proper to the Law citation model in the French speaking part of Belgium
- citation style not present yet into the citation style repository

### CODE REVIEW

**Comment by graulich** [comment]
Dear all,

we've got 3 issues from the csl bot and we are aware of the errors. All errors come from the file name and "self" link to zotero style repository.
Thank you for your comments !

File: ULiege-Droit.csl [hunk 1]
```diff
@@ -0,0 +1,520 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>ULiège - Droit (French)</title>
+    <title-short>ULiège-dt</title-short>
+    <id>http://www.zotero.org/styles/ULiege-Droit</id>
+    <link href="http://www.zotero.org/styles/universite-libre-de-bruxelles-histoire" rel="template"/>
```
Comment (damnation333) [inline ULiege-Droit.csl]: you need a self link here that matches the id and also has the same filename then.
See how it's done in your template style: https://github.com/citation-style-language/styles/blob/master/universite-de-liege-histoire.csl
It should be the full name. Universite the Liege - Droit (French) and then <id>http://www.zotero.org/styles/universite-de-liege-droit</id> and the same for the self link

Comment (rmzelle) [inline ULiege-Droit.csl]: @graulich, thanks for the submission. Before you do any more work, is there a style guide available for this citation format, either online or in print, that we could link to?

**Comment by rmzelle** [comment]
@graulich, we can only go forward with this if you answer my question at https://github.com/citation-style-language/styles/pull/2776#discussion_r124706156. Does the Université de Liège have a webpage where they describe this citation format? If not, how did you discover what the format should look like?

**Comment by graulich** [comment]
Thanks for your comments. We've made some modifications in accordance with your proposals and it has passed the csl bot test ! Could you tell us what are the next steps ? Thanks again.

**Comment by rmzelle** [comment]
@graulich, I need an answer to my earlier question. We only accept styles that are formally described (through either an online or printed style guide), so you need to convince us that that is the case here. See also https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link.

**Comment by damnation333** [comment]
Also, like your template style the filename, id and self-link should be fully written out as mentioned before: http://www.zotero.org/styles/universite-de-liege-droit

**Comment by graulich** [comment]
@damnation333 we are not opposed to fully write out our institution name in our template style (filename, id and self-link) as "universite-de-liege-droit" but the recent new branding of our institution is "ULiège" as "UMons", "UNamur", "UGent", etc.

**Comment by graulich** [comment]
@rmzelle We will shortly post a link to a formalised guide which explains our style (formally described style guide).

**Comment by rmzelle** [comment]
> We will shortly post a link to a formalised guide which explains our style (formally described style guide).

Great, thanks. We typically also rely the style guide to make sure we name the style correctly.

**Comment by graulich** [comment]
@rmzelle Here is the link to a formalised guide which explains our style: http://lib.ulg.ac.be/sites/default/files/guidedelafeuilledestyle.pdf (hosted on this platform http://lib.ulg.ac.be/fr/services/s-autoformer-en-ligne > tab "Tutoriels spécialisés". I've inserted it in the csl. Thanks again for your comments.

**Comment by rmzelle** [comment]
> we are not opposed to fully write out our institution name in our template style (filename, id and self-link) as "universite-de-liege-droit" but the recent new branding of our institution is "ULiège" as "UMons", "UNamur", "UGent", etc.

Just curious: is the "ULiège" branding very recent? When I visit e.g. http://www.ulg.ac.be it still says Université de Liège in several prominent places.

Also, would it be correct to name this style after the faculty ("Faculté de Droit, de Science politique et de Criminologie"), instead of the discipline ("Droit")? We typically would name a style like this "Université de Liège - Faculté de Droit, de Science politique et de Criminologie", if it's a faculty-wide used style.

**Comment by graulich** [comment]
@rmzelle 
Dear Rmzelle,
Indeed the name 'ULiège' is very recent. It was unveiled publicly in May 2017 but will not be effective until the beginning of the 2017-2018 academic year in October (new website, change of URL ...). Scientists are however invited by the university authorities to use the new name now to avoid further adaptations in October.
Our faculty is actually composed of three departments (law, political science and criminology), however the style developed is only conformed to the mode of citation used by lawyers (law department). The two other departments are considering the opportunity to develop a style appropriate to their discipline.
Thank you so much for the feedback,
I remain at your disposal.

**Comment by rmzelle** [comment]
How about  `<title>ULiège - Département de Droit (French)</title>`?

**Comment by graulich** [comment]
We would better accept the name "Université de Liège - Droit (French)". 
We can still use the full name of our institution as it will perhaps be more understandable to all. 
Nevertheless, considering the fact that in the speaking part of Belgium university departments are much more related to teaching matters than with research matters and law practice, we would prefer to stick with the "Droit" designation of the style. The style will probably be used mainly by fellow researchers from the law practising french speaking community of Belgium. Best regards / Cordialement / Met vriendelijke groeten.

**Comment by rmzelle** [comment]
Great, thanks!

**Comment by adam3smith** [comment]
That's great -- here are instructions for how to update your style (see 3b):
https://github.com/citation-style-language/styles/blob/master/CONTRIBUTING.md

**Comment by graulich** [comment]
Thanks, we will do that !

**Comment by rmzelle** [comment]
And you can find your current style at https://github.com/citation-style-language/styles/blob/master/universite-de-liege-droit.csl.

**Comment by graulich** [comment]
We have also post a style with some variation made to the "universite-de-liege-droit.csl" by following the procedure for the "dependent style" here: https://github.com/citation-style-language/styles/pull/3682. We are not so sure that it's the good instance for proposing a "variation" to our currently-beeing-updated-style (https://github.com/citation-style-language/styles/pull/3681). Could you help us ? The "variation" (universite-de-liege-droit-bibliographie.csl") consists of a classification of the bibliography, by document's type and then by author's name (alphabetical order). Thanks in advance for your answers and help.

**Pull request merged**